cea/technologies/thermal_network/thermal_network.py

```diff
@@ -2575,11 +2641,10 @@ def calc_supply_temperatures(t_ground__k, edge_node_df, mass_flow_df, k, t_targe
                     calc_t_out(i, edge, k, m_d, z, t_e_in, t_e_out, t_ground__k, z_note, thermal_network)
             plant_node = t_node.nonzero()[0]  # the node indices of the plant nodes in the edge-node index
 
-
             # Identify all nodes with no in or outflows and delete those values from the z matrixes
             # This is necessary to avoid getting stuck in a loop network with no mass flows inside the loop
             for i in range(z_note.shape[0]):
-                if np.isclose(sum(np.dot(m_d, z_pipe_out[i])),0) and np.isclose(sum(np.dot(m_d, z_pipe_in[i])),0):
+                if np.isclose(sum(np.dot(m_d, z_pipe_out[i])), 0) and np.isclose(sum(np.dot(m_d, z_pipe_in[i])), 0):
```

Comment:
actually, i think it's a good habit to use  `0.0` instead of  `0` to really drive home what we're talking about here (floats) as integers are a totally different beast.

Reply:
done